roles/gitops/rendering-apps-files/tasks/generate_dsc_json.yml

```
@@ -0,0 +1,33 @@
+---
+- name: Define template dsc.json.j2 path
+  ansible.builtin.set_fact:
+    json_template_path: "{{ role_path }}/config/dsc.json.j2"
+
+- name: Loop over apps
+  ansible.builtin.set_fact:
+    envs:
+      - name: "{{ dsc_name }}"
+        apps:
+          - argocd_app: keycloak
```

Il faudrait définir des valeurs de syncWave pour certains services différent du défaut.
```suggestion
          - argocd_app: keycloak
            syncWave: ?
```